packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -254,11 +255,25 @@ const ReflectionCard = (props: Props) => {
     !isComplete &&
     !isDemoRoute() &&
     (isHovering || !isDesktop)
+
+  const reflectionCount = reflectionGroups.reduce(
```

Comment:
#### Help Wanted
I am currently learning relay and graphql. When I use reflectionCount after adding the data dependency in the fragment container, its value is always 0, though reflections are present in the meeting.
So, I have manually calculated it.  
Any help what I am missing.

Reply:
You're not missing anything. `reflectionCount` is not calculated on the server side, but is just statically set when ending the meeting:
* it's read statically here: https://github.com/ParabolInc/parabol/blob/884ac2ef450a5f3f79c5da521c2caca9a3cc7c67/packages/server/graphql/types/RetrospectiveMeeting.ts#L73
* the source for `RetrospectiveMeeting`, i.e. the argument to the resolve function is the object loaded from rethink https://github.com/ParabolInc/parabol/blob/884ac2ef450a5f3f79c5da521c2caca9a3cc7c67/packages/server/database/types/MeetingRetrospective.ts#L33
* it's written in this mutation https://github.com/ParabolInc/parabol/blob/884ac2ef450a5f3f79c5da521c2caca9a3cc7c67/packages/server/graphql/mutations/endRetrospective.ts#L51

It would be possible to move this calculation to the server side, but your current approach is fine

Reply:
Ok, thanks @Dschoordsch!